bolt/lib/Core/DebugData.cpp

```diff
@@ -505,22 +511,18 @@ void DebugAddrWriterDwarf5::update(DIEBuilder &DIEBlder, DWARFUnit &CU) {
   DWARFDebugAddrTable AddrTable;
   DIDumpOptions DumpOpts;
   constexpr uint32_t HeaderSize = 8;
```

Comment:
Extract this to class level so you can also use it in DebugAddrWriterDwarf5::updateAddrBase instead of doing + 8.